src/Fujifilm/Connection/ConnectionManager.java

```diff
@@ -0,0 +1,29 @@
+package Fujifilm.Connection;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionManager {
+	static Connection conn;
+	static String url;
+
+	public static Connection getCustConnection() {
+
+		try {
+			url = "jdbc:mysql://127.0.0.1:3306/fujifilm?useSSL=false";
+			Class.forName("com.mysql.cj.jdbc.Driver");
+			conn = DriverManager.getConnection(url, "root", "1995");

```

Comment:
Please remove password. We can add code to pick it up from a local file later. For testing and development I am using a blank password for now.